packages/graphql-transformer-common/src/ResourceConstants.ts

```diff
@@ -29,16 +29,16 @@ export class ResourceConstants {
     OpenSearchStreamingLambdaFunctionLogicalID: 'OpenSearchStreamingLambdaFunction',
     OpenSearchDataSourceLogicalID: 'OpenSearchDataSource',
 
-    // RDS
-    RDSLambdaIAMRoleLogicalID: 'RDSLambdaIAMRole',
-    RDSLambdaLogAccessPolicy: 'RDSLambdaLogAccessPolicy',
-    RDSPatchingLambdaIAMRoleLogicalID: 'RDSPatchingLambdaIAMRole',
-    RDSLambdaLogicalID: 'RDSLambdaLogicalID',
-    RDSPatchingLambdaLogAccessPolicy: 'RDSPatchingLambdaLogAccessPolicy',
-    RDSPatchingLambdaLogicalID: 'RDSPatchingLambdaLogicalID',
-    RDSLambdaDataSourceLogicalID: 'RDSLambdaDataSource',
-    RDSLambdaDataSourceLogicalName: 'RDSLambdaDatabase',
```

Comment:
Note: I removed this constant -- it was not in use anywhere and was easily confused with `RDSLambdaDataSourceLogicalID`.